docs/integration/configuring-windows-agent.md

```diff
@@ -0,0 +1,41 @@
+title: Configuring the Sematext Windows Agent
+description: Tailor Sematext Windows Agent based on your needs to add/remove channels or to give friendly names to windows logs.
+
+The [Sematext Agent for Windows](https://sematext.com/docs/agents/sematext-agent/windows-installation/) collects logs from your Windows system and sends them to your [Windows Logs App](https://sematext.com/docs/integration/windows/#logs) for monitoring and analysis. After installing the agent, you can make additional configurations to tailor it to your preferences.
+
+## Configuring Event Names
+
+Windows logs are identified by an Event ID and a Source. For instance, **Event ID 4625** with **Source: Microsoft-Windows-Security-Auditing** signifies an unsuccessful logon attempt. You don’t need to memorize each Event ID and Source because the agent includes a built-in dictionary that translates these IDs and sources into more meaningful names. For example, when the agent encounters a log with **Event ID 4625** and **Source: Microsoft-Windows-Security-Auditing**, it assigns the name `ACCOUNT_LOGON_FAILED`.
+Although the agent provides a default dictionary, you can add or customize event names as needed by following these steps:
+
+1. Navigate to `C:\Program Files\SematextAgent\properties\` and open the `st-agent.yml` file.
+2. Locate the section labeled `#Define custom names for specific event IDs and sources`.
+3. Uncomment the `names`: line if it is commented out, and then add your custom event names. For example:
+
+```
+names:
+  "1033:MsiInstaller": "MSI_INSTALLATION_STARTED"
+  "6005:EventLog": "EVENT_LOG_SERVICE_STARTED"
+```
+4. Save changes and restart the Sematext Agent Service
+
+## Customizing Log Channels
+
+By default, the agent collects logs from the following Windows Event Viewer channels:
+
+- Security
+- Application
+- System
+- Setup
+
+You can customize which channels the agent collects logs from by following these steps:
+
+1. Navigate to `C:\Program Files\SematextAgent\properties\` and open the `st-agent.yml` file.
+2. Locate the section labeled paths and add or remove channels according to your needs.
+
+```
+paths: ["Security", "Application", "System", "Setup"]
```

Comment:
Are these 4 all available channels or are there more? If the latter, should we list them?

Reply:
These are the default ones. But people can add custom ones based on their configuration.

Reply:
How does one know which other ones are valid strings? What happens if one types FooBar here?

Reply:
The other channels they may want to add will be custom ones, which they've added to Windows Event Viewer later on. They should have the correct string from there. Assuming these are technical users, they will be aware of this. If the channel name is invalid or not found, we won't collect logs from it.

Reply:
If they use "FooBar" it breaks Windows Events collection, we show a log like this one;
```
time="2024-09-19T22:09:50+03:00" level=error msg="skip starting Windows Events Collector: failed to initialize eventlog: subscription of Windows Event Log failed: The specified channel could not be found." source="winevents/events_windows.go:81" visibility=user

```